arm-apimanagement/2016-10-10/swagger/apimusers.json

```diff
@@ -490,6 +490,59 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/users/{uid}/token": {
+      "post": {
+        "tags": [
+          "UserToken"
+        ],
+        "operationId": "Users_GetSharedAccessToken",
```

Comment:
`Noun` part of `operationId` should match tags by conventions. Please update `operationId` to `UserToken_GetSharedAccessToken` or change the `tags` to `Users`. Former on will create operation group name as `UserToken` while later one creates operation group name as `Users`. So make sure sure which one you prefer in SDK